openjdk8/pom.xml

```diff
@@ -81,7 +81,7 @@
 	    <configuration>
 	      <imageName>openjdk8</imageName>
```

Comment:
Yes,I don't think the image name necessarily has to match the directory structure of the code. The reason we wanted `openjdk` is to be able to provide a `openjdk:latest` tag for users who just want to be on the latest version of Java.